app/ui/src/app/connections/common/configuration/configuration.service.ts

```diff
@@ -45,7 +45,7 @@ export class ConnectionConfigurationService {
         break;
       default:
     }
-    const formModel = this.formFactory.createFormModel(
+    this.formFactory.createFormModel(
```

Comment:
You can actually delete this block I think, the next line is the same thing but with a return

Reply:
🤦‍♂️ I didn't notice that!